### PR TITLE
Add missing acknowledgements

### PR DESCRIPTION
**Pull request by mih**
In the absence of feedback on #8 I am adding a standalone sentence in
the style that I consider sensible for grants (was supported by + flat list).

### CODE REVIEW

**Comment by adswa** [comment]
I like the sentence, but shouldn't this information be one sentence up, together with the other grant information?

**Comment by mih** [comment]
Possibly. I was wondering about the desired structure in that issue. Once there is a consensus, things can be moved, I think.

File: paper.md [hunk 1]
```diff
@@ -346,6 +346,7 @@ through [CRCNS](https://www.nsf.gov/funding/pgm_summ.jsp?pims_id=5147) program.
...
 We express our gratitude to Joey Hess for the development and maintenance of git-annex, and for years of productive collaboration with the DataLad team.
+This development was supported by the European Regional Development Fund (Project: Center for Behavioral Brain Sciences Magdeburg, Imaging Platform), the European Union’s Horizon 2020 research and innovation programme under grant agreements [Human Brain Project (SGA3, H2020-EU.3.1.5.3, grant no. 945539)](https://cordis.europa.eu/project/id/945539), and [Virtual Brain Cloud (H2020-EU.3.1.5.3, grant no. 826421)](https://cordis.europa.eu/project/id/826421), the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under grants [SFB 1451 (431549029)](https://gepris.dfg.de/gepris/projekt/431549029) and [IRTG2150 (269953372)](https://gepris.dfg.de/gepris/projekt/269953372).
```
Comment (yarikoptic) [inline paper.md:349]: Should go one sentence before, so we have grants and then people. Otherwise Joey is getting lost above

Comment (mih) [reply paper.md:349]: I am fine with any position, as long as the numbers are in. I proposed a structure for this section. I could not see one.

**Pull request merged**